pypit/__init__.py

```diff
@@ -16,10 +16,7 @@
 import warnings
 
 # Set version
-__version__ = '0.8.0.dev0'
-# TODO: Do we need this?  A release should be tied to a date, and the
-# date information could be put in the CHANGES.rst
-__last_updated__ = '23Mar2018'
+__version__ = '0.8.0dev'
```

Comment:
Does this mean we are moving to 0.8 with this PR?

Or can that be deferred to a later PR named v0.8 ?

Reply:
Version `0.8.0dev` means that this is the development version that will end up being version `0.8.0` when we're satisfied.  When this PR gets merged, you can run:

```
bumpversion release --dry-run --verbose
```

to see that the release version will be `0.8.0`.